authorizer_lambda/tests/test_verify_cognito_token.py

```diff
@@ -0,0 +1,53 @@
+import json
+from datetime import datetime
+from pathlib import Path
+
+import jwt
+
+from authorizer_lambda import cognito_adapter
+from authorizer_lambda.config import config
+from authorizer_lambda.main import verify_cognito_token, get_owner_uuid
+from authorizer_lambda.models import CognitoResponse
+
+
+def test_return_owner_uuid_cognito(mocker):
+    mock_jwt_decode = mocker.patch.object(jwt, "decode")
+    mock_jwt_decode.return_value = {"sub": "a-b-c-d"}
+
+    request = Path(__file__).parent / Path("data/authorization_request.json")
+    request = json.load(request.open("rt"))
+    token = request["authorizationToken"]
+
+    response = verify_cognito_token(token)
+
+    expected = "ce4291d7-2604-41bb-ae61-c9cee3d38402"
+
+    assert response == expected
+    mock_jwt_decode.assert_called_once_with(token, config.public_rsa_key, algorithms=["RS256"])
+
+
+def test_fetch_owner_from_cognito(mocker):
+    mock_generate_cognito_response = mocker.patch.object(cognito_adapter, "fetch_owner_from_cognito")
```

Comment:
This test fails because you mock the wrong location. In python unittesting/mocking, you need to mock the location where the code is used, not where it is defined.  See https://docs.python.org/3/library/unittest.mock.html#where-to-patch

```suggestion
    mock_generate_cognito_response = mocker.patch.object(main, "fetch_owner_from_cognito")
```

Reply:
When I implement it that way, the test works fine.

Reply:
Oh I see, I thought it was the same as Jest for typescript. Thanks! I lost a good hour slamming my head into my keyboard trying to fix this issue haha.